components/tabs/index.en-US.md

```diff
@@ -45,3 +46,4 @@ Ant Design has 3 types of Tabs for different situations.
 | forceRender | Forced render of content in tabs, not lazy render after clicking on tabs | boolean | false |
 | key | TabPane's key | string | - |
 | tab | Show text in TabPane's head | string\|ReactNode | - |
+More option at [rc-tabs option](https://github.com/react-component/tabs#tabpane)
```

Comment:
Would you mind adding a blank line before this?

Reply:
no need to add blank line after table https://deploy-preview-15420--ant-design.netlify.com/components/tabs/#header

Reply:
We should keep markdown source clear and readable too.

Reply:
In the future maybe prettier for this?